src/api/app/views/layouts/webui/responsive_ux/_unread_notifications.html.haml

```diff
@@ -0,0 +1,5 @@
+= link_to(my_notifications_path, class: 'nav-link text-light p-0 w-100', alt: 'Notifications') do
+  %i.fas.fa-bell
+  - unless User.session.unread_notifications.zero?
+    %span.badge.badge-primary.align-text-top= User.session.unread_notifications
+  %span.d-block Notifications
```

Comment:
Add a newline here, please

Reply:
Yep, sorry. There was a PR merged recently and I didn't make rebase. Now the issue is gone. Thanks for the notice:)